<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configure</class>
 <widget class="QWidget" name="configure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>300</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>通道功能</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>303</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>通道功能</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>747</x>
     <y>440</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>551</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>通道设置</string>
   </property>
   <widget class="QComboBox" name="comboBox_chanFunc_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>通道号</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_Mode_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>440</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>正常</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>程序</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_default">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>660</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>77</x>
     <y>440</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>660</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_chanFunc">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>压力</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>峰值</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>变化率</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>气压计</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>100</y>
     <width>551</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>通道设置</string>
   </property>
   <widget class="QComboBox" name="comboBox_chanFunc_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>B</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>通道号</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox_chanFunc_2">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>300</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>压力</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>峰值</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>变化率</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>气压计</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_state">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>660</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>状态</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Mode">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>440</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>正常</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>程序</string>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(251, 255, 198);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>comboBox_Mode_2</zorder>
  <zorder>btn_default</zorder>
  <zorder>label_2</zorder>
  <zorder>btn_ok</zorder>
  <zorder>comboBox_chanFunc</zorder>
  <zorder>comboBox_chanFunc_2</zorder>
  <zorder>btn_state</zorder>
  <zorder>comboBox_Mode</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
